c/ntdrivers/parport.i.cil-2.c

```diff
@@ -4314,6 +4334,7 @@ NTSTATUS PptDeselectDevice(PVOID Context , PVOID DeselectCommand )
 }
 ULONG Ppt1284_3AssignAddress(PDEVICE_EXTENSION DeviceExtension ) 
 { UCHAR i = __VERIFIER_nondet_char() ;
+  __VERIFIER_assume(i < 7);
```

Comment:
In #1072  `__VERIFIER_assume` was replaces by implementation. Please replace it with `assume_cycle_if_not()` which is already defined for this program.

Reply:
Yes, `__VERIFIER_assume` has no special meaning anymore.